README.md

```diff
@@ -70,7 +69,10 @@ vim config/ganache.json
 git clone https://github.com/graphprotocol/graph-node/
 cd graph-node/docker
 
-# Run it
+# On Linux you may have to run
+./setup.sh
```

Comment:
Yep, sorry I saw this and didn't add it in the README

Can you say instead:
```
# [only linux] Inject host IP in docker-compose.yml
``

Reply:
Done